packages/sit-onyx/src/components/OnyxListboxOption/OnyxListboxOption.vue

```diff
@@ -0,0 +1,85 @@
+<script lang="ts" setup>
+import { createListbox } from "@sit-onyx/headless";
+import type { OnyxListboxOptionProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxListboxOptionProps>(), {
+  disabled: false,
+});
+
+const emit = defineEmits<{
+  "update:modelValue": [value: boolean];
+}>();
+
+const {
+  elements: { option },
+} = createListbox({
+  label: "",
+  onSelect: () => emit("update:modelValue", !props.modelValue),
```

Comment:
Can we pass in the data via props? I don't think it's a good idea to have a composable that needs to be used in two places to work.
There is an implicit dependency between the usage in the parent and in the option.
As soon as the composable has some kind of state, it will become an issue.

My suggestions from most preferable to the least preferable
1. Use the composable only in the parent and pass the attributes down via props
1. Inline this component into the parent
1. Create a separate composable for the options.

Reply:
I removed the composable from the option, can you check it again please?
I am not sure if there is a cleaner way to implement, if you find one, please let me know :)